.clang-format

```diff
@@ -138,7 +138,6 @@ ReflowComments:  true
 RemoveBracesLLVM: false
 SeparateDefinitionBlocks: Always
 ShortNamespaceLines: 1
-SortIncludes:    CaseSensitive
```

Comment:
The same key is present here https://github.com/riscv-software-src/riscv-perf-model/pull/92/commits/44291791647e5e562f4a1820be98555cb7f81668#diff-1026e0038b722990204a42bed8a6f7c0ec2302aa79e3fad1959d62ba968edfa2L192